projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.html

```diff
@@ -18,95 +23,124 @@
     value="{{ chosenWord }}"
   />
 
-  <cx-icon
-    [type]="iconTypes.RESET"
+  <button
     [attr.aria-label]="'common.reset' | cxTranslate"
     (mousedown)="clear(searchInput)"
     (keydown.enter)="clear(searchInput)"
     class="reset"
     tabindex="0"
-  ></cx-icon>
+  >
+    <cx-icon [type]="iconTypes.RESET"></cx-icon>
+  </button>
 
-  <cx-icon
-    [type]="iconTypes.SEARCH"
-    [attr.aria-label]="'common.search' | cxTranslate"
+  <div
```

Comment:
why no button?

Reply:
The Search button was pretty useless as you would only see it when the search input is empty. As soon as you start typing a search term, the search button is replaced by the reset button. 

The main reason it was removed is that it was confusing for Screen Reader users, as they could tab to it and upon hitting enter (to submit the search query) it would not perform any action since the search input is empty. 

The search button was converted to a non-interactive icon as it is only for ornamental (presentation only) use for sighted users.

Reply:
I agree with this. The so-called button didn't trigger any event, neither with mouse or keyboard. That magnifying glass was always meant to be ornamental. However, a small CSS adjustment is needed to remove the pointer on hover.

Reply:
pointer removed on `hover`